### PR TITLE
Crm 18204

### PR DESCRIPTION
**Pull request by pradpnayak**

### CODE REVIEW

File: CRM/UF/Page/Field.php [hunk 1]
```diff
@@ -126,7 +126,9 @@ public function browse() {
     $this->assign('isGroupReserved', $isGroupReserved);
 
     $profileType = CRM_Core_BAO_UFField::getProfileType($this->_gid);
-    if ($profileType == 'Contribution' || $profileType == 'Membership' || $profileType == 'Activity' || $profileType == 'Participant') {
```
Comment (yashodha) [inline CRM/UF/Page/Field.php]: You might wanna use _checkProfileType_ method for checking whether the profile is mixed or not.

**Comment by pradpnayak** [comment]
Thanks @yashodha 

I have modified code to add the changes

**Comment by pradpnayak** [comment]
jenkins, retest please

**Comment by pradpnayak** [comment]
jenkins, retest this please

**Pull request merged**